README.md

```diff
@@ -149,15 +149,15 @@ cargo-fmt
 
 ## Contributing
 
-Please see our [contributing guidlines](https://github.com/Joystream/joystream#contribute) for details on our code of conduct, and the process for submitting pull requests to us.
+Please see our [contributing guidlines](./pioneer/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
```

Comment:
The link was broken because we renamed our repos. We will place a CONTRIBUTING.md file at the root, so please make it link to `./CONTRIBUTING.md` instead.